lib/menubar/init.lua

```diff
@@ -132,6 +132,7 @@ local function load_count_table()
                 count_table[name] = count
             end
         end
+        count_file:close()
     end
```

Comment:
The whole file handling here could be replaced with `for line in io.lines(filename) do` (and that function even automatically closes the file when it hits its end, so much simpler!)